Tests/XCUITests/PhotonActionSheetTests.swift

```diff
@@ -67,6 +67,8 @@ class PhotonActionSheetTests: BaseTestCase {
 
         // This is not ideal but only way to get the element on iPhone 8
         // for iPhone 11, that would be boundBy: 2
+        mozWaitForElementToExist(app.otherElements["ActivityListView"].otherElements["Example Domain"])
+        mozWaitForElementToExist(app.otherElements["ActivityListView"].otherElements["example.com"])
```

Comment:
Sometimes, the fennec button does not do anything but exists. 😐 The `ActivityListView` may not be displayed completely....yet!